packages/jellyfish-transaction-builder/src/txn/txn_builder_masternode.ts

```diff
@@ -41,4 +47,90 @@ export class TxnBuilderMasternode extends P2WPKHTxnBuilder {
       changeScript
     )
   }
+
+  /**
+   * Build update masternode transaction
+   *
+   * @param {UpdateMasternode} updateMasternode transaction to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @param {Object} [collateral] needed when updating owner address
+   * @param {string} collateral.txid collateral txid
+   * @param {BigNumber} collateral.value collateral amount
+   * @param {number} collateral.tokenId collateral tokenId
+   * @param {Script} collateral.newOwnerScript for new owner address
+   * @return {Promise<TransactionSegWit>}
+   */
+  async update (
+    updateMasternode: UpdateMasternode,
+    changeScript: Script,
+    collateral?: {
+      txid: string
+      value: BigNumber
+      tokenId: number
+      newOwnerScript: Script
+    }
+  ): Promise<TransactionSegWit> {
+    const minFee = new BigNumber(0.001)
+    const { prevouts, vin, total } = await this.collectPrevouts(minFee)
+
+    const deFiOut: Vout = {
+      value: new BigNumber(0),
+      script: {
+        stack: [
+          OP_CODES.OP_RETURN,
+          OP_CODES.OP_DEFI_TX_UPDATE_MASTER_NODE(updateMasternode)
+        ]
+      },
+      tokenId: 0x00
+    }
+
+    const change: Vout = {
+      value: total,
+      script: changeScript,
+      tokenId: 0x00
+    }
+
+    const mergedVin = [...vin]
+    const mergedVout = [deFiOut, change]
+    const mergedPrevouts = [...prevouts]
+
+    if (collateral !== null && collateral !== undefined) {
+      const { txid, value, tokenId, newOwnerScript } = collateral
+
+      const collateralPrevout: Prevout = {
+        txid,
+        vout: 1,
+        script: changeScript,
+        value: new BigNumber(value),
+        tokenId
```

Comment:
```suggestion
        tokenId: 0x00
```

So we can just use `0x00`.